src/OrchardCore.Modules/OrchardCore.Media/Permissions.cs

```diff
@@ -37,12 +36,10 @@ public IEnumerable<PermissionStereotype> GetDefaultStereotypes()
                 new PermissionStereotype
                 {
                     Name = "Author",
-                    Permissions = new[] { ManageOwnMedia }
```

Comment:
I think this will prevent these two roles from being able to select media from the media library when editing content.

Can you test and verify?

If that is the case, then they'll need the `ManageMedia` permission by default

Reply:
> If that is the case, then they'll need the ManageMedia permission by default

I think that too, but in this case all non Administrator will be equivalent, let me test and verify ...

Reply:
@deanmarcussen this will not work either because of this check

https://github.com/OrchardCMS/OrchardCore/blob/e6aa8e4f3a42d79c7bd2627b622cd792163cb37b/src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs#L77-L80

I'm wondering how it works before for non Administrator permission?!!